core/metadata.js

```diff
@@ -63,7 +63,11 @@ const log = logger({
 
 const { platform } = process
 
-const CONFLICT_REGEXP = /-conflict-\d{4}(?:-\d{2}){2}T(?:\d{2}_?){3}\.\d{3}Z/
+const DATE_REGEXP = '\\d{4}(?:-\\d{2}){2}T(?:\\d{2}_?){3}\\.\\d{3}Z'
+const SEPARATOR_REGEXP = `(?!.*\\${path.sep}.*)`
```

Comment:
I'd have written this as `[^\\${path_sep}]*$`, but both versions should work

Reply:
I remember trying this version (I'm not particularly familiar with the look-ahead) but it wouldn't work for some reason.
It could be a syntax error.